.github/workflows/ci.yml

```diff
@@ -1,37 +1,23 @@
 name: Continuous Integration
 
 on:
-  push:
   pull_request:
 
 jobs:
-  htmlproofer:
-    name: HTMLProofer
+  link-check:
+    name: Link Check
     runs-on: ubuntu-latest
     
     steps:
       - name: Checkout EIP Repository
         uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
-      
-      - name: Install OpenSSL
-        run: sudo apt-get update && sudo apt-get install -y libcurl4-openssl-dev
-    
-      - name: Install Ruby
-        uses: ruby/setup-ruby@08245253a76fa4d1e459b7809579c62bd9eb718a
-        with:
-          ruby-version: '2.6.0'
-          bundler-cache: true
         
-      - name: Build Website
-        run: |
-          bundle exec jekyll doctor
-          bundle exec jekyll build
-      
-      - name: HTML Proofer
-        run: bundle exec htmlproofer ./_site --check-html --check-opengraph --report-missing-names --log-level=:debug --assume-extension --empty-alt-ignore --timeframe=6w --disable-external
-      
-      - name: DNS Validator
-        run: bundle exec github-pages health-check
+      - name: Link Checker
```

Comment:
I guess one benefit to checking after the rendering instead of before is that any weirdness with Jekyll will be caught (see #5158 for an example of weirdness.) I'm not opposed to switching though, just wanted to raise the concern.

You've checked this against the repo already, I assume?

Reply:
> You've checked this against the repo already, I assume?

Yes. I even had to submit a PR to fix a small bug lol.

Reply:
> any weirdness with Jekyll will be caught

#5158 didn't get caught by HTMLProofer, though!

Reply:
It wasn't a link!